bundles/org.openhab.binding.philipsair/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="philipsair"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="pwr">
+		<item-type>Switch</item-type>
+		<label>Power</label>
+		<description>Device power</description>
+	</channel-type>
+	<channel-type id="om">
+		<item-type>String</item-type>
+		<label>Fan Speed</label>
+		<description>Fan speed</description>
+		<state pattern="%s">
+			<options>
+				<option value="s">Silent</option>
+				<option value="1">1</option>
+				<option value="2">2</option>
+				<option value="3">3</option>
+				<option value="t">Turbo</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="mode">
+		<item-type>String</item-type>
+		<label>Mode</label>
+		<description>Filter mode</description>
+		<state pattern="%s">
+			<options>
+				<option value="P">Auto</option>
+				<option value="A">Allergen</option>
+				<option value="S">Sleep</option>
+				<option value="M">Manual</option>
+				<option value="B">Bacteria</option>
+				<option value="N">Night</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="uil">
+		<item-type>Switch</item-type>
+		<label>Buttons</label>
+		<description>Buttons light</description>
+		<state pattern="%s" />
+	</channel-type>
+	<channel-type id="ddp">
+		<item-type>String</item-type>
+		<label>Displayed index</label>
+		<description>Display PM2.5 index or Allergen Index</description>
+		<state readOnly="false" pattern="%s">
+			<options>
+				<option value="1">PM2.5</option>
+				<option value="0">Allergen</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="aqil">
+		<item-type>Number</item-type>
+		<label>LED light level</label>
+		<description>Level 100, 75, 50, 25, 0</description>
+		<state pattern="%d" min="0" max="100" step="25" />
+	</channel-type>
+	<channel-type id="cl">
+		<item-type>Switch</item-type>
+		<label>Child lock</label>
+		<description>Child lock enabled/disabled</description>
+	</channel-type>
+
+	<channel-type id="dtrs">
+		<item-type>Number</item-type>
+		<label>Timer left</label>
+		<description>Timer minutes lefts</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>

```

Comment:
Please apply openHab's [Units of Measurement](https://www.openhab.org/docs/concepts/units-of-measurement.html) framework wherever applicable. This allows user interfaces greater flexibility when it comes to rendering data from your binding.
```suggestion
	<channel-type id="dtrs">
		<item-type>Number:Time</item-type>
		<label>Timer left</label>
		<description>Timer minutes lefts</description>
		<state readOnly="true" pattern="%d" />
	</channel-type>

```